src/Microsoft.AspNetCore.Build/scripts/KoreBuild.sh

```diff
@@ -125,6 +124,18 @@ ensure_msbuild
 
 KOREBUILD_TARGETS_ROOT="$KOREBUILD_ROOT/src/Microsoft.AspNetCore.Build/targets"
 
+MSBUILD_RSP=$(mktemp -t koremsbuild)
+
+cat > $MSBUILD_RSP <<ENDMSBUILDARGS
```

Comment:
Not technically, but it's a little easier to keep track of arguments if they are the same format in both, IMO.